util/Migrator/DbScripts/2023-05-16_00_ClientSecretHash.sql

```diff
@@ -0,0 +1,80 @@
+IF COL_LENGTH('[dbo].[ApiKey]', 'ClientSecretHash') IS NULL
+BEGIN
+  ALTER TABLE [dbo].[ApiKey]
+  ADD [ClientSecretHash] VARCHAR(128);
+END
+GO
+
+-- Refresh views
+IF OBJECT_ID('[dbo].[ApiKeyDetailsView]') IS NOT NULL
+    BEGIN
+        EXECUTE sp_refreshview N'[dbo].[ApiKeyDetailsView]';
+    END
+GO
+
+IF OBJECT_ID('[dbo].[ApiKeyView]') IS NOT NULL
+    BEGIN
+        EXECUTE sp_refreshview N'[dbo].[ApiKeyView]';
+    END
+GO
+
+-- Drop existing SPROC
+IF OBJECT_ID('[dbo].[ApiKey_Create]') IS NOT NULL
+BEGIN
+    DROP PROCEDURE [dbo].[ApiKey_Create]
+END
+GO
+
+-- Create the new SPROC
+CREATE PROCEDURE [dbo].[ApiKey_Create]
```

Comment:
Use `CREATE OR ALTER`

Reply:
resolved in https://github.com/bitwarden/server/pull/2943/commits/d3b2069012d03d9799ec5bd8087cd1e9633593bc